### PR TITLE
Adds a False Testemony charge to Marine Law

### PR DESCRIPTION
**Pull request by Smov1**
# About the pull request

 Adds a new Marine Law charge "False Testemony" to the JAS system. It was classified as a minor crime and 5 minutes brig timer.

# Explain why it's good for the game

Right now, Marine Law doesn't cover people purposely lying to MPs when they are asked about whether they saw who did what, etc. The closest thing we have is aiding and abetting but that usually only applies when someone actively attempts to help a suspect.

If the person that lied to the MP is proved to have lied, Marine Law should cover this circumstance. 

# Testing Photographs and Procedure

image

# Changelog

:cl: smov
add: Adds a new Marine Law charge
/:cl:

### CODE REVIEW

**Comment by morrowwolf** [comment]
Get this approved by Forest.

File: code/modules/law/laws/minor_crime.dm [hunk 1]
```diff
@@ -50,3 +50,8 @@
 	desc = "To take items (or property) from another person or entity without their express permission, or to retain possession of items that have been taken without permission. This includes the removal of vendors from department areas without permission."
 	special_punishment = "Return of Stolen Item"
 	brig_time = 10
+
+/datum/law/minor_law/false_testemony
+	name = "False Testemony"
```
Comment (realforest2001) [inline code/modules/law/laws/minor_crime.dm:55]:
```suggestion
/datum/law/minor_law/false_testimony
	name = "False Testimony"
```

File: code/modules/law/laws/minor_crime.dm [hunk 2]
```diff
@@ -50,3 +50,8 @@
...
+	desc = "To intentionally and willingly lie to a MP or deputized officer, during a legal investigation with the intent to disrupt said investigation."
```
Comment (realforest2001) [inline code/modules/law/laws/minor_crime.dm:56]:
```suggestion
	desc = "To intentionally and willingly lie to an MP, or deputized officer, during a legal investigation with the intent to disrupt said investigation."
```

File: code/modules/law/laws/minor_crime.dm [hunk 3]
```diff
@@ -50,3 +50,8 @@
 	desc = "To take items (or property) from another person or entity without their express permission, or to retain possession of items that have been taken without permission. This includes the removal of vendors from department areas without permission."
 	special_punishment = "Return of Stolen Item"
 	brig_time = 10
+
+/datum/law/minor_law/false_testemony
```
Comment (realforest2001) [inline code/modules/law/laws/minor_crime.dm:54]: I'm fine with this as a law.

**Review by realforest2001 — CHANGES_REQUESTED** [review]

File: code/modules/law/laws/minor_crime.dm (hunk 2 above)
Comment (Diegoflores31) [reply code/modules/law/laws/minor_crime.dm:56]: Miranda rights 💀

**Comment by morrowwolf** [comment]
Make sure to update the ML section on the wiki (or pass it to wiki maintainers at least) when this is merged.

**Review by harryob — APPROVED** [review]

**Pull request merged**